### PR TITLE
Test/addresses controller

### PR DESCRIPTION
**Pull request by PedroTerriquez**

### CODE REVIEW

File: spec/controllers/addresses_controller_spec.rb [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+  end
+
+  it 'renders the edit addresses view' do
+    get :edit, params: { id: address }
+    expect(response).to be_ok
+  end
+
+  it 'create a new address' do
+    post :create, params: { address: {
+      alias: 'test',
```
Comment (lacf95) [inline spec/controllers/addresses_controller_spec.rb]: DRY

**Review by lacf95 — CHANGES_REQUESTED** [review]
Litttle changes :)

File: spec/controllers/addresses_controller_spec.rb (hunk 1 above)
Comment (lacf95) [reply spec/controllers/addresses_controller_spec.rb]: Salu2

Comment (PedroTerriquez) [reply spec/controllers/addresses_controller_spec.rb]: Thanks!! 👍

**Review by IsaiasAntonio — APPROVED** [review]

**Review by lacf95 — APPROVED** [review]

**Pull request merged**